content/adr/drupal-incremental-updates.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Incremental updates to Drupal"
+date: "2024-02-29"
```

Comment:
Why this date?

Reply:
Added this based on [when we discussed about this](https://axelerant.slack.com/archives/C06LU6DDVEX/p1709182475884979?thread_ts=1709039712.881909&cid=C06LU6DDVEX) when doing the CDM packages